components/cmp/modal/src/index.scss

```diff
@@ -151,6 +151,7 @@ $lh-sui-cmp-modal-title: $lh-xxl !default;
     font-size: $fz-sui-cmp-modal-inner;
     overflow-y: scroll;
     padding: $p-l;
+    min-height: 50vh; // fix for IE
```

Comment:
From my point of view, it's a really bad practice to make things for IE, but well, it's already merged :(

Reply:
I would say the same with complicated supports but I think that the tradeoff is good.
It gives a lot of value with only 1 line IMO @kikoruiz